src/organization/configuration/user/new/org-settings-new-user.component.html

```diff
@@ -61,7 +65,7 @@ <h2>Identity Provider</h2>
     </mat-card-content>
 
     <mat-card-actions align="end">
-      <button mat-button color="accent" type="submit" [disabled]="userForm.invalid">Save</button>
+      <button mat-button color="accent" type="submit" [disabled]="(userForm.statusChanges | async) !== 'VALID'">Save</button>
```

Comment:
🤯  to avoid ExpressionChangedAfterItHasBeenCheckedError error with `userForm.invalid` 

I found this issue where they did the same workaround as me https://github.com/angular/angular/issues/18004